todo.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable func-names */
+/* eslint-disable no-plusplus */
+
+import './checkbox';
+
+let myToDo = [];
+
+if (localStorage.myToDo !== undefined) {
+  myToDo = JSON.parse(localStorage.myToDo);
+}
```

Comment:
Hello Vanessa! Your Project is dry, and I don't see problems with scalability. According to the KISS best practice principle, you could do a short circuit evaluation here, so I suggest using something like this: `let myToDo = JSON.parse(localStorage.myTodo) || [];`